.github/actions/e2e/install-karpenter/action.yaml

```diff
@@ -15,6 +15,9 @@ inputs:
     required: true
   git_ref:
     description: "The git commit, tag, or branch to check out. Requires a corresponding Karpenter snapshot release"
+  ecr_repo:
```

Comment:
Any reason we need to pass the `ecr_repo` value through here as an input? I don't see a place in the workflows or actions where this value changes.

Reply:
Not really, I wanted a constant since it was referenced in a couple of places but this probably isn't the way to do it. Is there a better way or would it be better to just hardcode it in at both locations?